posts/2021-11-03-tensorflow-gpu.rst

```diff
@@ -41,8 +41,15 @@ You can now select between GPU enabled (default) and CPU packages using the `ten
     # OR
     conda install tensorflow-gpu -c conda-forge
 
-When installing the `tensorflow` package, the package resolution will now default to the `tensorflow-gpu` package (that will also work on CPU). If you want the slimmer "cpu-only" package, then you can install `tensorflow-cpu` directly.
+When installing the `tensorflow` package, the package resolution will now default to the `tensorflow-gpu` package if the local machine has a GPU. Note that GPU-enabled packages can also work on CPUs, but one would need to override the enviornment variable `CONDA_CUDA_OVERRIDE` like below. This could be handy if you are in a situation where your current node (e.g. login node) on an HPC does not have GPUs, but the compute nodes with GPUs do not have interenet access.
 
+.. code-block:: bash
+
+    CONDA_CUDA_OVERRIDE="" mamba install tensorflow -c conda-forge
```

Comment:
This is not recommended as this would give the user "some GPU" package instead of the most performant one possible for their GPU.

Reply:
Should it be "11.2" only?

Reply:
Yes. It should be either 10.2, 11.0, 11.1 or 11.2. Whatever the cuda version the driver installed supports.

Reply:
Or should we explicitly instruct the user to select something they think is best for them?

Reply:
Okay, let me edit it then, thanks!